### PR TITLE
Added new file

### PR DESCRIPTION
**Pull request by diwassapkota805**
Can you check if the added file is correct

### CODE REVIEW

File: newFile.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+I added this text (diwassapkota805)
+
```
Comment (diwassapkota805) [inline newFile.txt:2]: can you see this?

Comment (diwassapkota80) [inline newFile.txt:2]: put something in this line

**Review by diwassapkota805 — COMMENTED** [review]
hiiii

File: newFile.txt (hunk 1 above)
Comment (diwassapkota805) [inline newFile.txt:2]: i can't see your comment

Comment (diwassapkota80) [reply newFile.txt:2]: How about now?

**Comment by diwassapkota80** [comment]
YOU MOST BE ABLE TO SEE THIS - diwassapkota80

File: newFile.txt (hunk 1 above)
Comment (diwassapkota805) [inline newFile.txt:2]: reply to this comment please

Comment (diwassapkota80) [inline newFile.txt:2]: cann you seee thissss?

Comment (diwassapkota80) [reply newFile.txt:2]: yes

**Review by diwassapkota80 — APPROVED** [review]
good job

**Pull request merged**